### DISCUSSION TITLE
Kanalinfos

### DISCUSSION THREAD

**Discussion by Sebastian-Voigt** [Xyaren/homeassistant-magentatv#215] [Ideas]
Ist es möglich den Sendernamen und die Laufende Sendung auszulesen?

**Comment by Xyaren** [comment]
Sendernamen sind meines Wissens nach nicht möglich da der Receiver diese Information nicht zur Verfügung stellt.
Jedoch ist die Kanalnummer unter dem Attribut `media_channel` abrufbar.
Die Laufende Sendung ist unter `media_title` verfügbar.

![image](https://github.com/user-attachments/assets/ffe13d09-d177-4fc5-8248-ce066bd5faf2)

**Reply by Sebastian-Voigt** [reply]
Danke für die Antwort, nach einer Weile konnte ich das Attribut media_title auch finden. 
Dies tauchte allerdings erst nach 1-2h auf. 
Also alles super.
Vielen Dank für die Integration. ;-)

**Reply by Xyaren** [reply]
Leider ist die Integration abhänging von Events die der Receiver sendet.
Diese kommen teilweise nur wenn sich das laufende Programm ändert oder man den Kanal wechselt.
